src/functionalTest/java/uk/gov/hmcts/reform/sscscorbackend/GetAnOnlineHearingTest.java

```diff
@@ -1,23 +1,59 @@
 package uk.gov.hmcts.reform.sscscorbackend;
 
+import static org.hamcrest.CoreMatchers.nullValue;
 import static org.hamcrest.core.Is.is;
 import static org.junit.Assert.assertThat;
 
 import java.io.IOException;
-import java.util.UUID;
 import org.json.JSONObject;
 import org.junit.Test;
 
 public class GetAnOnlineHearingTest extends BaseFunctionTest {
+    private final String decisionAward = "FINAL";
+    private final String decisionHeader = "The decision";
+    private final String decisionReason = "Decision reason";
+    private final String decisionText = "The decision text";
+
     @Test
     public void getAnOnlineHearing() throws IOException {
-        String emailAddress = "someEmail" + UUID.randomUUID().toString() + "@example.com";
+        int randomNumber = (int) (Math.random() * 10000000);
+        String emailAddress = "test" + String.valueOf(randomNumber) + "@hmcts.net";
```

Comment:
Its a tiny point but "test" + randomNumber will result in a string so you don't need String.valueOf here